tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3204,6 +3204,31 @@ def test_transaction_user_misery(self) -> None:
             },
         }
 
+    def test_sorting(self) -> None:
+        resp = self._on_demand_query_check(
+            {
+                "field": ["user_misery(300)", "apdex(300)"],
+                "project": self.project.id,
+                "query": "",
+                "sort": "-user_misery(300)",
```

Comment:
We can probably just switch another test with multiple specs to check order by in the opposite direction instead of re-inserting a bunch of rows for a new test.

Reply:
I haven't read the rest of the tests but might be easier to just have a setUp with a spec & data that can be shared amongst most tests but 🤷 

I'll just add some more tests for now